travis/main.sh

```diff
@@ -40,6 +38,14 @@ check_git_status() {
 build_autotools() {
 	v --fold="bootstrap" ./bootstrap.sh
 
+	case ${TRAVIS_BUILD_TYPE} in
+	sanitizers)
+		SANITIZE_FLAGS="-fsanitize=address -fsanitize=leak -fsanitize=alignment -fsanitize=bool -fsanitize=enum -fsanitize=float-cast-overflow -fsanitize=float-divide-by-zero -fsanitize=function -fsanitize=integer-divide-by-zero -fsanitize=null -fsanitize=object-size -fsanitize=return -fsanitize=signed-integer-overflow -fsanitize=unreachable -fsanitize=unsigned-integer-overflow"
```

Comment:
this line is waaaaaaaay too long :).  use an array instead:

```
SANITIZE_FLAGS=(
  -fsanitize=address
  -fsanitize=leak
  -fsanitize=alignment
  -fsanitize=bool
  ....
)
CFLAGS+=" ${SANITIZE_FLAGS[*]}"
```

since CFLAGS will be unset by default, this won't run with optimizations.  so you'll want to add a helper func like `set_default_compiler_settings` (not set on the name) which would do:

```
set_default_compiler_settings() {
  : ${CFLAGS=-O2 -pipe}
  export CFLAGS
}
```